pnpm-lock.yaml

```diff
@@ -7,6 +7,11 @@ overrides:
 
 packageExtensionsChecksum: 60d08949101fc004c7564f98037b7764
 
+patchedDependencies:
+  meaning-of-life@1.0.0:
+    hash: o3deharooos255qt5xdujc3cuq
+    path: examples/npm_deps/patches/meaning-of-life@1.0.0-pnpm.patch
+
```

Comment:
When you use `pnpm.patchedDependencies` you get this entry in the lockfile. It's an index of all of the patches.